### ISSUE TITLE
Some Questions

### ISSUE THREAD

**Issue by Diklabyte** [MightyPirates/OpenComputers#1799]
First of all, I'm complete new on GitHub. Please have patience, if I make something wong or similar.

Well, i have two questions:
1. **Touch Events**
   If you bind a Tier 1 GPU and a Tier 2 or 3 Screen, I can check the Touch signals in my Code.
   Is this wanted or a Bug? I think this was not in the past?
2. **Unicode/Glyphs**
   First of all: The new Font 'unscii' is beautiful! But the Font has not all Glyphs as 'Unifont'.
   My code uses some _special_ Glyphs such as Arrows or a Clock Sign and much more.
   But the new Version with the new Font doesn't have these Glyphs anymore.
   Will these coming back in the future, or not?
   If not, can I do something, to get these back?
   It's about these Glyphs:
   `0x21C5` = Up/Down-Arrow,
   `0x21B9` = Tab-Arrow,
   `0x23CE` = Enter-Arrow,
   `0x21C4` = Left/Right-Arrow,
   `0x2191` = Arrow-Up,
   `0x2193` = Arrow-Down,
   `0x23F2` = Clock Glyph.
   (I'm not sure about the Arrow-Up or Arrow-Down Glyph, but I think one of these is implemented)

Any help are appreciated, Thanks :)

**Comment by MyNameIsKodos** [comment]
Touch events are tied to the screen tier, not the GPU.  So yes, a T2+ Screen will feed screen events to your code.

As for the font, I _think_ it's a config option

**Comment by Vexatos** [comment]
For what it's worth, you are free to help complete the font at https://github.com/asiekierka/unscii-asie.

**Comment by Diklabyte** [comment]
@MyNameIsKodos Ahh, ok that is new for me.. Nevermind, my code need an Update for that. Ok thank you so much!
No the config part is only for Texture-Rendering (without Unicode), and hexfont (with Unicode) settings. So unfortunately I can't fix this by changing config.

@Vexatos Well, the thing is, I made the Glyphs before, but the new 'wcwidth.bin' file ignored them completely.
The Version before #1760 the glyphs are rendered as expected.
But the width on the Glyphs are incompatible, so this throws me a Lua-exception.
The Version after that, I have no Problems anymore, except that the glyphs renders me an '?' for a placeholder.
The console says me, that the glyphs will be ignored.
With or without I have the same count on loaded glyphs.
I tried this with 8x16 and 8x8, doesn't really matter.
Well, sure I can try to _update_ the 'wcwidth.bin', but this is the only thing where I have no idea how to do this.
